### PR TITLE
fix displayed upload size: divide by 2 since we have 2 files

### PR DESCRIPTION
**Pull request by davidak**
tested template with http://jinja2test.tk/

please test deployment

### CODE REVIEW

**Comment by lamby** [comment]
Thanks!  I might try fixing this the other way around so  we aren't doing maths in this file and rather in the presentation level.

**Comment by davidak** [comment]
I choose this file so i have to change only one line.

If it makes more sense i can do it in https://github.com/lamby/trydiffoscope/blob/master/templates/compare/view.html

**Comment by lamby** [comment]
> If it makes more sense i can do it in

I think so, yes. Feel free to rename the variable (eg. ``..PER_FILE`` or ``..TOTAL`` etc. etc., whatever makes more sense)

**Comment by lamby** [comment]
@davidak Any update on this? :)

**Comment by davidak** [comment]
@lamby i just changed it. sorry i forgot it.

File: templates/compare/view.html [hunk 1]
```diff
@@ -50,7 +50,7 @@
                   <div class="form-group {% if form.errors.file_a %}has-error{% endif %}">
                     <label for="{{ form.file_a.id_for_label }}">
                       File #1
-                      <span class="text-muted">(max: {{ settings.TRYDIFFOSCOPE_MAX_UPLOAD_SIZE_MEGABYTES }}MB)</span>
+                      <span class="text-muted">(max: {{ settings.TRYDIFFOSCOPE_MAX_UPLOAD_SIZE_MEGABYTES_TOTAL // 2 }}MB)</span>
```
Comment (lamby) [inline templates/compare/view.html]: This is Jinja-specific syntax. We are using Django templates.

Comment (davidak) [inline templates/compare/view.html]: Ah, ok.

Is `{% widthratio settings.TRYDIFFOSCOPE_MAX_UPLOAD_SIZE_MEGABYTES_TOTAL 2 1 %}` the right way? Or do i need to put the variable between `{{}}`?

I found http://stackoverflow.com/questions/8447913/is-there-a-filter-for-divide-for-django-template

**Comment by lamby** [comment]
@davidak 

Fixed in 4f8e13316a2926a2f0519c9630aea4a83b9cb3d1

Not sure why you didn't just clone the repo and test yourself...? :)

**Comment by davidak** [comment]
@lamby i wanted to avoid setting up this whole project just for such simple modification. sorry if i wasted our time :(

**Comment by lamby** [comment]
@davidak I'm just being selfish - if you had setup the project (really quite easy!) I could have got more contributions out of you ;-)

**Comment by davidak** [comment]
@lamby that's true, but my biggest problem is the lack of time and i'm contributing to a lot of projects.

**Pull request closed without merging**